src/components/mainPage/upload.css

```diff
@@ -29,3 +40,22 @@
 .cloud-img {
   padding-bottom: 0.8rem;
 }
+
+.file-container{
+  background-color: #efefef;
+  border: 0.1rem solid black; 
+  border-radius: 0.1rem;
+  padding: 1rem 1rem;
+  text-indent: 1.3rem;
+  font-size: 1.125rem;
+  font-weight: 34.375rem;
```

Comment:
font-weight doesn't need to use rem. font-weight is unitless https://www.w3.org/TR/CSS1/#font-weight

Reply:
Ok, thanks for the feedback. I've fixed the filenames issue, also added margin space at end of the list. Is there anything else I should modify before I start resolving the merge conflicts.